axelor-purchase/src/main/java/com/axelor/apps/purchase/service/PurchaseOrderLineService.java

```diff
@@ -104,4 +105,6 @@ public void checkDifferentSupplier(
 
   public List<PurchaseOrderLine> updateLinesAfterFiscalPositionChange(PurchaseOrder purchaseOrder)
       throws AxelorException;
+
+  LocalDate getEstimatedReceiptDate(PurchaseOrderLine purchaseOrderLine, Partner supplierPartner);
```

Comment:
Hello,

Please may you return a Optional<LocalDate> which is better suited when the return value can be null